docs/description-of-used-coding-standards-rules.md

```diff
@@ -115,18 +115,13 @@ You can see [official documentation](https://phpmd.org/rules/index.html) for mor
 
 | Rule name                                           | Rule description                                                                                                                   | Note                                         |
 | ---------------------------------------------       | ---------------------------------------------------------------------------------------------------------------------------------- | ----                                         |
-| `rulesets/codesize.xml/CyclomaticComplexity`        | Cyclomatic complexity of a method must not be greater than 13.                                                                     | Can not be fixed automatically.              |
```

Comment:
To be honest, I am not really sure if the file has adequate value. It basically duplicates the information already present in different files (something like `SELECT name, description FROM our_ruleset JOIN tool_documentation`) but we already have it and that means we should maintain it.
You removed mentions of PHPMD rules but did not add appropriate PHP-CS-Fixer rules. Can you please fix it?

Reply:
> To be honest, I am not really sure if this file has an adequate value. It basically duplicates the information already present in different files

I agree it has barely any value. Rules are already clear from configs, they should be commented there if you need to add something extra, like [this in this config](https://github.com/Symplify/Symplify/blob/master/packages/EasyCodingStandard/config/clean-code.neon).

Also, you package should document **your code**. This is 3rd party code that is already covered on their side.

**I suggest removing this one and only document your extra Sniffs/Fixers.**

Reply:
I agree. Lets keep only the information that cannot be found somewhere else. You did a good job removing the file completely in https://github.com/shopsys/coding-standards/pull/6/commits/ac9212c5c5517efada4a59780f10265fb6b59514.

The new texts in the README will need some minor fixes but I will do it after we merge this. It's a pity that I as a reviewer cannot somehow add commits into your PR - that would make it much easier.